velox/vector/LazyVector.h

```diff
@@ -122,6 +122,9 @@ void setRunTimeStatWriter(std::unique_ptr<BaseRuntimeStatWriter>&& ptr);
 // ever be accessed, loading can be limited to these positions. This
 // also allows pushing down computation into loading a column, hence
 // bypassing materialization into a vector.
+// Unloaded LazyVectors should be referenced only by one top-level vector.
+// Otherwise, it runs the risk of being loaded for different set of rows by each
```

Comment:
> it runs the risk of being loaded for different set of rows by each

I wonder if this is clear enough to the reader. The problem is that second load will be ignored and if the rows specified in the second load are different from the rows in the first load, then some of the rows will be unavailable causing failures down the line.